package.json

```diff
@@ -73,6 +73,11 @@
         "vite": "^5.4.0",
         "vitest": "^1.6.0"
     },
+    "pnpm": {
+        "overrides": {
+            "rollup": "^4.22.4"
+        }
+    },
```

Comment:
why is this needed?

Reply:
Pnpm audit fails due to a vulnerability in rollup which is a dependency on vite. vite hasn't released a fix yet so this takes care of the audit for the time being.